django/db/models/fields/related.py

```diff
@@ -1731,8 +1731,13 @@ def deconstruct(self):
         # Rel needs more work.
         if isinstance(self.remote_field.model, str):
             kwargs["to"] = self.remote_field.model
+            if "." in self.remote_field.model:
+                app_label, model_name = self.remote_field.model.split(".")
+                kwargs["to"] = "%s.%s" % (app_label, model_name.lower())
+            else:
+                kwargs["to"] = self.remote_field.model.lower()
```

Comment:
ln1733 looks redundant. `kwargs["to"]` is reassigned on either branch here. 🤔

Reply:
Yes, should be removed :facepalm: